common/src/main/resources/minecraft_access-common.mixins.json

```diff
@@ -4,7 +4,8 @@
   "package": "com.github.khanshoaib3.minecraft_access.mixin",
   "compatibilityLevel": "JAVA_17",
   "mixins": [
```

Comment:
Just a small refactor, these are better to be moved to "client" section

Reply:
I've noticed that, but I can't find a valiable field for extracting the mob name from pure-client-side classes.

Reply:
> Just a small refactor, these are better to be moved to "client" section

I've noticed that, but I can't find a valiable field for extracting the mob name from pure-client-side classes.
Oh, you mean we can simply move the two to `client` list without any error?

Reply:
Yup, that should not cause any error and ensure that the mod is client sided.